scripts/build.py

```diff
@@ -648,7 +724,19 @@ def generate_build_directory():
     save_hashes_as_json(HASHES_JSON, hashes)
 
     # Minify all template files copy them into build/templates/head.
-    build_files(TEMPLATES_DEV_DIR_CORE, TEMPLATES_STAGING_DIR, hashes)
+    if not os.path.isdir(TEMPLATES_STAGING_DIR):
```

Comment:
Let's add more explanation. Maybe a comment that's something like "If there is no staging dir, perform the build process on all files."

Reply:
More comments have been added to the building process of `/dev/templates/`